src/mlpack/core/data/imputation_methods/custom_imputation.hpp

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file custom_imputation.hpp
+ * @author Keon Kim
+ *
+ * Definition and Implementation of the empty CustomImputation class.
+ */
+#ifndef MLPACK_CORE_DATA_IMPUTE_STRATEGIES_CUSTOM_IMPUTATION_HPP
+#define MLPACK_CORE_DATA_IMPUTE_STRATEGIES_CUSTOM_IMPUTATION_HPP
+
+#include <mlpack/core.hpp>
+#include <cmath>
+
+using namespace std;
```

Comment:
We should avoid `using` declarations in header files, even though it might make the rest of the code in this file longer, because then the using declaration applies to every single file that includes this header, and that may not be what the user wants.